keys/employees-keys/README.md

```diff
@@ -116,10 +133,160 @@ sub   4096R/CC8BE1D5 2021-04-02
 
 The key ID in this case is `7429663E`.
 
-## How to export your key
+### Sending key for signing
+
+Generate ASCII armor public key file:
 
 ```
 gpg --armor --output your-name-key.asc --export "YOUR KEY ID HERE"
 ```
 
-Share `your-name-key.asc` textual file as the public portion of your key.
+Obtain fingerprint of you key:
+
+```
+gpg -n -q --import --import-options import-show your-name-key.asc
+```
+
+Armored public key should be sent to
+[3mdeb chat web-of-trust channel](https://chat.3mdeb.com/team-3mdeb/channels/web-of-trust).
+
+Please send fingerprint to 3mdeb Team Leaders or Management using different
+channel then 3mdeb chat e.g. https://keybase.io, LinkedIn, email, etc.
+Fingerprint should be sent to person who is work with you on signing your keys.
```

Comment:
@pietrushnic `who is working` or `who works`

Reply:
```suggestion
Fingerprints should be sent to the person who works with you on signing your keys.
```